buildCoreClr.ps1

```diff
@@ -42,15 +43,14 @@ elseif ($Configuration -match 'PSv3') {
 
 if ($build)
 {
-
     Write-Progress "Building Engine"
     Push-Location Engine\
-    dotnet build Engine.csproj --framework $Framework --configuration $Configuration
+    $null = dotnet build Engine.csproj --framework $Framework --configuration $Configuration
```

Comment:
As long as issue #913 is not resolved and we do not have 'warning as error' as a build option, don't you think it would make sense to see the build output?

Reply:
I'm ok with that - my preference is usually to see nothing, but not all share that view.